apisix/plugins/authz-keycloak.lua

```diff
@@ -53,10 +54,24 @@ local schema = {
         keepalive_timeout = {type = "integer", minimum = 1000, default = 60000},
         keepalive_pool = {type = "integer", minimum = 1, default = 5},
         ssl_verify = {type = "boolean", default = true},
+        client_id = {type = "string", minLength = 1, maxLength = 100},
+        client_secret = {type = "string", minLength = 1, maxLength = 100},
+        lazy_load_paths = {type = "boolean", default = false},
+        http_method_as_scope = {type = "boolean", default = false},
     },
+    required = {"client_id"},
```

Comment:
To avoid break change, we should not make `client_id` required. If `client_id` missed, we should continue the old logic.

Reply:
done